### PR TITLE
a9-float layout

### PR DESCRIPTION
**Pull request by Juanmediina**
Creacion de actividad a9 float-layout

### CODE REVIEW

File: src/jm/html-css/style.css [hunk 1]
```diff
@@ -0,0 +1,130 @@
+body {
+  background-image: url(./image/2.png);
```
Comment (Iregui44) [inline src/jm/html-css/style.css:2]: I dont see the use of float anywhere

File: src/jm/html-css/index.html [hunk 2]
```diff
@@ -0,0 +1,83 @@
+<!doctype html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
```
Comment (Iregui44) [inline src/jm/html-css/index.html:5]:
```suggestion
    <meta name="viewport" content="width=device-width, initial-scale=1.0">
```

File: src/jm/html-css/index.html [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+            <li>
+              <a href="#"><i class="fa-solid fa-clapperboard"></i>Records</a>
+            </li>
+            <li>
+              <a href="#"
+                ><i class="fa-solid fa-folder-plus"></i>Create Playlist</a
+              >
+            </li>
+            <hr />
+            <li><a href="#">My playlist #1</a></li>
```
Comment (Iregui44) [inline src/jm/html-css/index.html:66]: I think you are missing a Ul tag here

**Review by Iregui44 — CHANGES_REQUESTED** [review]

File: src/jm/html-css/style.css (hunk 1 above)
Comment (Juanmediina) [reply src/jm/html-css/style.css:2]: use of float has already been added

File: src/jm/html-css/index.html (hunk 2 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html]: corrected

File: src/jm/html-css/index.html (hunk 3 above)
Comment (Juanmediina) [reply src/jm/html-css/index.html:68]: UI has been added

**Review by Iregui44 — APPROVED** [review]

**Pull request merged**